DataConnectors/ZPA/Connector_LogAnalytics_agent_Zscaler_ZPA.json

```diff
@@ -0,0 +1,166 @@
+{
+    "id": "ZPA",
```

Comment:
Can you please expand the id? "zscalarprivateaccess" is a suggestion.

Reply:
updated